src/components/index/SectionLayout.tsx

```diff
@@ -0,0 +1,11 @@
+interface SectionLayoutProps {
```

Comment:
I think we should group `SectionLayout` and `SectionHeading` into `/section` folder